### ISSUE TITLE
item visual bug

### ISSUE THREAD

**Issue by Ltntech** [PrinceOfAmber/Storage-Network#437]
Minecraft Version: 1.18.1

Forge Version: 39.0.88

Mod Version: 1.6.0

Single Player or Server: Single player

Describe problem (what you were doing; what happened; what should have happened):Hello I put 64 item in the chest and in storage inventory it show 128 item

Log file link:I don't have log file

Video/images/gifs (direct upload or link): https://www.mediafire.com/file/28z4x8iqondaoae/2022-03-28+13-30-20.mov/file

**Comment by Ltntech** [comment]
Please fix

**Comment by HalcyonSeas73** [comment]
That can happen when your chest is connected twice somewhere. Fix your cabling.

**Comment by Lothrazar** [comment]
Yes if two cables connect to the same inventory it will show the items twice, nothing is duped if you interact.

**Comment by Ltntech** [comment]
ok
